src/App.vue

```diff
@@ -581,8 +566,8 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
     // since username is unique, use it as the user key
     const key: string = userInfo.username
     const email: string = userInfo.contacts[0]?.email || userInfo.email
-    const firstName: string = userInfo?.firstname
-    const lastName: string = userInfo?.lastname
```

Comment:
If it wasn't needed on the other userInfo properties then it's not needed here either.